javascript/javascript2/week1/animalNameGenerator.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SpiritAnimalNameGenarator</title>
+</head>
+<body>
+    <form>
+        <label for="name">Name of the user:</label>
+        <input type="text" name="Name" id="txtName" placeholder="Name"><br>
+        
+<input type="button" value="Clickme" id="btnSubmit" onclick="getUserName()">
```

Comment:
here you put an `onclick` event and refer to the function `getUserName`, can you also do that with the `addEventListener` functionality?